code/game/objects/noose.dm

```diff
@@ -0,0 +1,153 @@
+/obj/item/stack/cable_coil/building_checks(datum/stack_recipe/R, multiplier)
+	if(R.title == "noose")
```

Comment:
Bad.

Reply:
Not sure what exactly about this section of code is bad
`/obj/item/stack/proc/building_checks(datum/stack_recipe/R, multiplier)` is used in stack.dm already and this just specifies the cable_coil checks
and `if(R.title =="noose")` seems alright to me for defining the noose recipe

Reply:
String comparisons are slow. Check if `result_type` is the noose typepath instead.

Reply:
`if(R.result_type(noose)) ?`
cant find many references to `result_type` in the code, only in stacks.dm

Reply:
ahh cool, Will fo that now